doc/quickstart.rst

```diff
@@ -18,10 +18,249 @@ examples below:
 
 Live Reports are a key part of IOTile Analytics.  They are static HTML documents
 that either embed data and plots or reference local data files that can be
 accessed without needing a web server.
 
+.. important::
+
+	Download :download:`freezer.hdf5 <data/freezer.hdf5>` to use as source data and
+	place it in your current working directory.
+
+The quickest way to generate a live report is to use the `analytics-host` command
+line program that comes with `iotile-analytics-interactive`.  You can use the 
+program to generate reports from datasets that are either stored locally on your
+computer or downloaded on the fly from iotile.cloud.
+
+Almost always, the output of `analytics-host` is an interactive html+javascript
+document embedding graphs, data tables and other widgets to let you scroll through
+your data.  Sometimes the output is just plain text or a different binary format.
+
+Python packages you have installed can register LiveReport types with
+`analytics-host` so that you run them from the command line.  Let's see what
+reports you have installed:
+
+.. code:: bash
+
+	$ analytics-host -l
+	Installed Report Count: 3
+
+	Known Reports:
+	 - save_hdf5: Save all data locally as an HDF5 database file.
+	 - stream_overview: Display histogram and time series information about a stream.
+	 - basic_info: A basic LiveReport that just prints source information.
+
+
+You have 3 reports installed.  You can download data from iotile.cloud for fast local 
+access, you can print basic_info about a data source and you can create an interactive
+html report about a data stream.
+
+Let's look at the data source `freezer.hdf5` that you downloaded.
+
+.. code:: bash
+
+	$ analytics-host -t basic_info ./freezer.hdf5 -c
+	Source Info
+	-----------
+	active                         1
+	busy                           0
+	claimed_by                     arch-example-user
+	claimed_on                     2017-08-17T18:45:41Z
+	created_on                     2017-01-11T03:39:14Z
+	external_id
+	gid                            0000-0000-0000-00fd
+	id                             253
+	label                          Freezer Temperature
+	lat                            0.0
+	lon                            0.0
+	org                            arch-example-data
+	project                        05612d90-afc5-47be-b6a6-304b26c56488
+	sg                             single-temp-sensor-v1-1-0
+	slug                           d--0000-0000-0000-00fd
+	state                          N1
+	template                       pod-1gg1-es1-v2-0-0
+
+	Properties
+	----------
+
+Here we see that our downloaded data comes from a random device that is supposed to
+measure temperature and was named 'Freezer Temperature'.  Note that we always pass the `-c`
+parameter so that we don't have to confirm all of our actions.  Otherwise you will be
+prompted before actually generating a report.
+
+You can also see the data streams that the device has by giving an argument to the report.
```

Comment:
Not sure if you explain later, but may want to mention that the `-a` argument is report specific. i.e. defined by each report, so each report will have a different set.